src/functions.rs

```diff
@@ -128,6 +128,8 @@ impl Functions {
         functions.add(Identifier::from("plus"), stdlib::math::Plus);
         // string functions
         functions.add(Identifier::from("replace"), stdlib::string::Replace);
+        // list functions
+        functions.add(Identifier::from("is-empty"), stdlib::list::IsEmpty);
```

Comment:
Presumably there should also be a call that adds the `length` function?

Reply:
Yes, there should!